template/turf_2017/index.js

```diff
@@ -47,8 +47,10 @@ module.exports = function (comments, options, callback) {
         return hljs.fixMarkup(string);
       },
       getNpmPath: function (filepath) {
-        var module = filepath.match(/packages\/turf-([a-z-\d]+)/i)[1];
-        return "npm install @turf/" + module;
+        var folder = path.dirname(filepath);
+        var foldersArray = folder.split("\\");
+        var module = foldersArray[foldersArray.length-1].replace("turf-", "");
+        return "npm install @turf/" + module
```

Comment:
This section was changed on purpose, lets use Regex or some kind of "No OS specific" solution for this, on a Mac or Linux this doesn't work (`folder.split('\\')` won't work).

We just need to update the Regex express to fit both OS or use the `path` module more efficiently.